src/components/pages/doc.yml

```diff
@@ -7,3 +7,6 @@ permalinkCouldNotBeCopied:
 documentMetadataUpdated:
   en: Document updated successfully
   de: Das Dokument wurde erfolgreich aktualisiert
+commentsPanelTitle:
+  en: Viewing document page in "comments" mode
+  de: Viewing document page in "comments" mode
```

Comment:
I would like your help here, Andreas 🙏

Reply:
Maybe "Seitenansicht im Kommentar-Modus" or - as a full sentence: "Du befindest dich im Kommentar-Modus" (which would kinda nicely fit the close button on the right).

Reply:
Sounds good, went with the second option, thanks!